### DISCUSSION TITLE
5.x 版本 proxy 默认的8080端口，可以自定义吗

### DISCUSSION THREAD

**Discussion by xuanskyer** [apache/rocketmq#7174] [Q&A]
5.x 版本 proxy 默认的8080端口，可以自定义吗

**Comment by MaplePc** [comment]
config in file `conf/rmq-proxy.json`
and add option after command mqbroker `-pc $ROCKETMQ_HOME/conf/rmq-proxy.json`

**Comment by MaplePc** [comment]
There is an item `remotingListenPort` in that json file

**Reply by MmringZee** [reply]
Where can I see what parameters are configurable?

**Comment by MaplePc** [comment]
It looks like there's no document to refer, and&nbsp;my file is below:{
&nbsp; "rocketMQClusterName": "DefaultCluster",
&nbsp; "grpcServerPort": 8091,
&nbsp; "remotingListenPort": 8090
}

You can edit it in need.

&nbsp;


..
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年1月18日(星期四) 下午3:03
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [apache/rocketmq] 5.x 版本 proxy 默认的8080端口，可以自定义吗 (Discussion #7174)





 
Where can I see what parameters are configurable?

**Comment by MaplePc** [comment]
It looks like there's no document to refer, and my file is below:
```
{
  "rocketMQClusterName": "DefaultCluster",
  "grpcServerPort": 8091,
  "remotingListenPort": 8090
}
```
You can edit it in need.
